tests/nnstreamer_converter/runTest.sh

```diff
@@ -1,5 +1,32 @@
 #!/usr/bin/env bash
-source ../testAPI.sh
+
```

Comment:
Could you write a doxygen tag  in this PR  to generate doxygen document for a shell script as well as  *.c source code?  I think that now is appropriate time to append doxygen tags becasue the  ssat scripts are stable compared to the existing facility. :) 

* Reference: 
https://github.com/nnsuite/TAOS-CI/blob/master/ci/doc/doxygen-documentation.md#case-study-bash

Reply:
Done. Applied the standard SSAT template.